### DISCUSSION TITLE
Snow depth

### DISCUSSION THREAD

**Discussion by mesochaser2425** [niels747/2D-Weather-Sandbox#59] [Ideas]
So as the title says, my idea would be to add snow depth to the game. it could be great to se where snow has been falling more and just how deep is the snow, it could also affect melting speed, so couple hoter days would not melt all of the snow

**Comment by Meso22** [comment]
Weather stations will be added soon so I think you will be able to measure snow depth using those

**Reply by niels747** [reply]
I will add that
